optaweb-employee-rostering-backend/src/main/resources/org/optaweb/employeerostering/service/solver/employeeRosteringScoreRules.drl

```diff
@@ -279,6 +279,13 @@ end
 // Soft constraints
 // ############################################################################
 
+rule "Change of Employee for Published Shift"
+    when
+        Shift(moved == true)
+    then
+        scoreHolder.penalize(kcontext);
+end
+
```

Comment:
I'm missing a `ConstraintConfiguration` for this one.

Reply:
Technically not implemented yet, will remove until we implement non-distruptive planning (so not in scope)